src/angularComponent/ngcOmniboxController.js

```diff
@@ -240,10 +242,14 @@ export default class NgcOmniboxController {
    * @param {Object} item
    */
   unchoose(item) {
-    if (item && Array.isArray(this.ngModel)) {
-      this.ngModel.splice(this.ngModel.indexOf(item), 1);
-    } else if (!this.multiple) {
-      this.ngModel = null;
+    if (item) {
+      if (Array.isArray(this.ngModel)) {
+        this.ngModel.splice(this.ngModel.indexOf(item), 1);
```

Comment:
What happens if the item isn't in the array of selections? Does this splicing still work? i.e. if I pass a bogus argument to this method?